src/components/flowchart/index.js

```diff
@@ -391,7 +391,7 @@ export class FlowChart extends Component {
     const ENTER = 13;
     const ESCAPE = 27;
     if (event.keyCode === ENTER) {
-      this.handleNodeClick(node);
+      this.onLoadNodeData(node);
```

Comment:
I don't think this line should have been renamed - `onToggleNodeClicked` is different from `handleNodeClick`. This will cause issues and is ~likely~ breaking the keydown behaviour.
![image](https://user-images.githubusercontent.com/1155816/97994713-7e5e0180-1ddd-11eb-9adb-33fbc09a04ba.png)

Reply:
We should add some tests to ensure that this issue does not occur again in future

Reply:
yes indeed, great spot - we should definitely look into adding some tests for `handleNodeKeyDown` to ensure it won't break.